clickstream/src/test/java/software/aws/solution/clickstream/AutoRecordEventClientTest.java

```diff
@@ -198,7 +197,7 @@ public void testViewOneScreenEvent() throws Exception {
                     Assert.assertFalse(attributes.has(ReservedAttribute.PREVIOUS_SCREEN_NAME));
                     Assert.assertFalse(attributes.has(ReservedAttribute.PREVIOUS_SCREEN_ID));
                     Assert.assertEquals(1, attributes.getInt(ReservedAttribute.ENTRANCES));
-                    Assert.assertTrue(attributes.has(ReservedAttribute.ENGAGEMENT_TIMESTAMP));
+                    Assert.assertFalse(attributes.has(ReservedAttribute.ENGAGEMENT_TIMESTAMP));
```

Comment:
Do we have another test case to make sure this event triggered properly?

Reply:
Yes, this [test case](https://github.com/awslabs/clickstream-android/blob/fix_engagement_time/clickstream/src/test/java/software/aws/solution/clickstream/AutoRecordEventClientTest.java#L298) verifies that the `_engagement_time_msec` attribute is included in subsequent `_screen_view` events